halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/spin/v1/InstallSpinCommand.java

```diff
@@ -45,8 +43,8 @@
   String version = "latest";
 
   @Override
-  protected OperationHandler<RemoteAction> getRemoteAction() {
-    return new OperationHandler<RemoteAction>()
+  protected OperationHandler<String> getRemoteAction() {
+    return new OperationHandler<String>()
```

Comment:
Is this really desirable? This makes automation either copy-paste or piping into bash

Reply:
The argument here is that not every `spin` user will necessarily have `hal` installed since they won't necessarily be an admin. It seems unintuitive to need `hal` to acquire `spin`. We can go back to the `RemoteAction` plan, but then we'll need to support another path for users not using `hal` which also is annoying from a maintenance perspective.

Reply:
That's true, but isn't that why we have https://www.spinnaker.io/guides/spin/cli/? 

Also, if we do want this, rather than introduce different behavior by default, can we put this behind `--print-script-only`?